packages/react/src/components/Dropdown/Dropdown.test.tsx

```diff
@@ -290,262 +243,241 @@ describe('Dropdown', () => {
-      dropdownRoot.simulate('keydown', { which: KeyCodes.up });
-      expect(titleElement.text()).toEqual('3');
+      const dropdownRoot = getByRole('combobox');
+      const titleElement = within(dropdownRoot).getByText('3');
+      expect(titleElement).toBeTruthy();
+
+      userEvent.type(dropdownRoot, '{arrowdown}', { skipClick: true });
+      expect(titleElement.textContent).toEqual('3');
+      userEvent.type(dropdownRoot, '{arrowup}', { skipClick: true });
+      expect(titleElement.textContent).toEqual('3');
     });
 
     it('Keypresses on a normal dropdown selects the right, valid items.', () => {
       const options = [...DEFAULT_OPTIONS];
       options[3] = { key: 3, text: '3', selected: true };
-      wrapper = mount(<Dropdown label="testgroup" options={options} />);
-      const dropdownRoot = wrapper.find('.ms-Dropdown');
+      const { getByRole } = render(<Dropdown options={options} />);
+
+      userEvent.tab();
+      const dropdownRoot = getByRole('combobox');
+      const titleElement = within(dropdownRoot).getByText('3');
+      expect(titleElement).toBeTruthy();
 
-      const titleElement = dropdownRoot.find('.ms-Dropdown-title');
-      expect(titleElement.text()).toEqual('3');
-      dropdownRoot.simulate('keydown', { which: KeyCodes.down });
-      expect(titleElement.text()).toEqual('4');
-      dropdownRoot.simulate('keydown', { which: KeyCodes.up });
-      expect(titleElement.text()).toEqual('3');
-      dropdownRoot.simulate('keydown', { which: KeyCodes.up });
-      expect(titleElement.text()).toEqual('2');
+      userEvent.type(dropdownRoot, '{arrowdown}', { skipClick: true });
+      expect(titleElement.textContent).toEqual('4');
+      userEvent.type(dropdownRoot, '{arrowup}', { skipClick: true });
+      expect(titleElement.textContent).toEqual('3');
+      userEvent.type(dropdownRoot, '{arrowup}', { skipClick: true });
+      expect(titleElement.textContent).toEqual('2');
     });
 
     it('does not select any item on focus', () => {
-      wrapper = mount(<Dropdown label="testgroup" options={DEFAULT_OPTIONS} />);
+      const { getByRole } = render(<Dropdown options={DEFAULT_OPTIONS} />);
 
-      wrapper.find('.ms-Dropdown').simulate('focus');
+      userEvent.tab();
 
-      const titleElement = wrapper.find('.ms-Dropdown-title');
-      expect(titleElement.text()).toEqual('');
+      const dropdownRoot = getByRole('combobox');
+      expect(document.activeElement).toBe(dropdownRoot);
+      expect(dropdownRoot.querySelector('.ms-Dropdown-title')?.textContent).toBe('');
     });
 
     it('can be programmatically focused when tabIndex=-1, and will not automatically select an item', () => {
       const dropdown = React.createRef<IDropdown>();
 
-      const container = document.createElement('div');
-      document.body.appendChild(container);
-
       // in enzyme, when we call the programmatic focus(), it does not trigger the onFocus callback of the div
```

Comment:
Nit: do we still need this comment since this PR eliminates Enzyme from this test suite?

Reply:
Maybe we need to update it with a link to [focus/blur](https://testing-library.com/docs/guide-events#focusblur) in the testing-library docs.

Reply:
Good catch. The comment doesn't even seem in line with the implementation prior to this PR, so I removed it entirely.